portal/pages/api/track.js

```diff
@@ -37,10 +46,18 @@ const handler = async (req, res) => {
 };
 
 function maskPhoneNumber(array) {
-  const obj = array[0];
-  let { phone_number } = obj;
-  phone_number = `******${phone_number.slice(6)}`;
-  obj.phone_number = phone_number;
+  let modifiyData = array.map(element => element.delivery_status);
```

Comment:
We shouldn't perform these additional functions (adding name and other attributes to the response) in the `maskPhoneNumber` function - can we create a separate function to do this and let `maskPhoneNumber` remain as it was

Reply:
Typo: `modifiyData` -> `modifyData`